fluentui_drawer/src/main/java/com/microsoft/fluentui/tokenized/bottomsheet/BottomSheet.kt

```diff
@@ -490,13 +512,32 @@ private fun Modifier.bottomSheetSwipeable(
                             "if the whole content is visible in Shown state itself"
                 )
             }
-            Modifier.swipeable(
-                state = sheetState,
-                anchors = anchors,
-                orientation = Orientation.Vertical,
-                enabled = sheetState.currentValue != BottomSheetValue.Hidden,
-                resistance = null
-            )
+            if (stickyBehavior) {
+                Modifier.swipeable(
+                    state = sheetState,
+                    anchors = anchors,
+                    orientation = Orientation.Vertical,
+                    enabled = sheetState.currentValue != BottomSheetValue.Hidden,
+                    thresholds = { from, to ->
+                        val fromKey = anchors.entries.firstOrNull { it.value == from }?.key
+                        val toKey = anchors.entries.firstOrNull { it.value == to }?.key
+                        if (fromKey != null && toKey != null && fromKey < toKey) {
+                            FixedThreshold(stickyThresholdDownward!!.dp) // Threshold for drag down
+                        } else {
+                            FixedThreshold(stickyThresholdUpward!!.dp)  // Threshold for drag up
```

Comment:
is this line expected to execute when either fromKey or toKey is null?

Reply:
In case of null it will default it to 56.dp, commited this new change